### PR TITLE
Documentation added for Sentinel-1 derived products and processing guide

### PR DESCRIPTION
**Pull request by megha-devaraju**

### CODE REVIEW

File: Data/Sentinel1_processing_guide.qmd [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    - Copernicus 30m DEM
+    - Copernicus 90m DEM 
+
+## [CloudFerro processing options](https://documentation.dataspace.copernicus.eu/APIs/On-Demand%20Production%20API.html) 
+
+CloudFerro offers CARD-BS and COH6/COH12 products as a [service on demand](https://documentation.dataspace.copernicus.eu/APIs/On-Demand%20Production%20API.html): 
+
+- [Sentinel-1 (CARD-BS) BackScatter](https://documentation.dataspace.copernicus.eu/Data/Additional.html#sentinel-1-card-bs)
+
+    - This processing option contains gamma0 geometric terrain correction (orthorectification), which is identical to the gamma0 (ellipsoid) backscatter coefficient with orthorectification option in Sentinel Hub processing options. No Radiometric Terrain Correction (RTC) is applied to this product.
```
Comment (Kampen1988) [inline Data/Sentinel1_processing_guide.qmd:48]: Please add a bullet list item "Additional information" that refers to https://creodias.eu/eodata/sentinel-1/sentinel-1-l2-backscatter-bs/

**Review by Kampen1988 — CHANGES_REQUESTED** [review]

File: Data/Sentinel1_processing_guide.qmd [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+format:
+  html:
+    code-fold: true
+jupyter: python3
+---
+
+Sentinel-1 data can be accessed and processed in different ways within Copernicus Data Space Ecosystem. Below we have compiled an overview of all the options to help you decide which one to use.
+
+## [Sentinel-1 RTC](https://documentation.dataspace.copernicus.eu/Data/Additional.html#sentinel-1-rtc) 
+[Sentinel-1 RTC](https://documentation.dataspace.copernicus.eu/Data/Additional.html#sentinel-1-rtc) (Radiometric Terrain Correction) SAR Backscatter is a product processed from Sentinel-1 GRD data and compliant with [CEOS Analysis Ready Data for Land (CARD4L) specifications](https://ceos-dev.ceos.org/ard/) for Normalised Radar Backscatter (NRB) products. Orthorectification is based on Copernicus DEM and no speckle filtering is applied. ([Additional product information](https://sentinels.copernicus.eu/web/sentinel/sentinel-1-ard-normalised-radar-backscatter-nrb-product))
```
Comment (Kampen1988) [inline Data/Sentinel1_processing_guide.qmd:12]: Please add `{target="_blank"}` behind links to external websites. (Point 2 in [Basic guidelines document](https://github.com/eu-cdse/documentation/blob/publish/EU-CDSE_Documentation_Guidelines_v0.pdf))

File: Data/Sentinel1_processing_guide.qmd [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+    - beta0
+    - sigma0
+    - gamma0 (ellipsoid)
+    - gamma0 (terrain) &rarr; **gamma0 RTC can only be performed if orthorectification is enabled**
+
+- [Lee Speckle Filtering](https://documentation.dataspace.copernicus.eu/APIs/SentinelHub/Data/S1GRD.html#processing-options) applied on source data after calibration and noise removal
+- [Orthorectification](https://documentation.dataspace.copernicus.eu/APIs/SentinelHub/Data/S1GRD.html#processing-options) with Range-Doppler terrain correction using one of the following DEMs:
+
+    - Copernicus 10m/30m DEM (10m resolution inside [39 European states](https://spacedata.copernicus.eu/collections/copernicus-digital-elevation-model) including islands and 30m elsewhere.)
```
Comment (Kampen1988) [inline Data/Sentinel1_processing_guide.qmd:38]: Please add `{target="_blank"}`

File: Data/Sentinel1_processing_guide.qmd [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    - Copernicus 90m DEM 
+
+## [CloudFerro processing options](https://documentation.dataspace.copernicus.eu/APIs/On-Demand%20Production%20API.html) 
+
+CloudFerro offers CARD-BS and COH6/COH12 products as a [service on demand](https://documentation.dataspace.copernicus.eu/APIs/On-Demand%20Production%20API.html): 
+
+- [Sentinel-1 (CARD-BS) BackScatter](https://documentation.dataspace.copernicus.eu/Data/Additional.html#sentinel-1-card-bs)
+
+    - This processing option contains gamma0 geometric terrain correction (orthorectification), which is identical to the gamma0 (ellipsoid) backscatter coefficient with orthorectification option in Sentinel Hub processing options. No Radiometric Terrain Correction (RTC) is applied to this product. 
+    - [Additional information](https://creodias.eu/eodata/sentinel-1/sentinel-1-l2-backscatter-bs/).
```
Comment (Kampen1988) [inline Data/Sentinel1_processing_guide.qmd:49]: Please add `{target="_blank"}`

File: Data/Sentinel1_processing_guide.qmd [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+- [Sentinel-1 (CARD-COH) Coherence](https://documentation.dataspace.copernicus.eu/APIs/On-Demand%20Production%20API.html)
+
+    - The Sentinel-1 CARD COH (Copernicus Analysis Ready Data Coherence) processor generates a Sentinel-1 Level 2 product describing the coherence of a pair of images - 12 days apart. Concurrently, a terrain-correction (using DEM) is performed. 
+    - [Additional information](https://creodias.eu/eodata/sentinel-1/sentinel-1-l3-bs-monthly-com/).
```
Comment (Kampen1988) [inline Data/Sentinel1_processing_guide.qmd:54]: Please add `{target="_blank"}`

File: Data/Sentinel1_processing_guide.qmd [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+## openEO processing options 
+
+openEO provides the **[sar_backscatter()](https://processes.openeo.org/draft/#sar_backscatter)** method to process, calibrate and normalise Sentinel-1 radar backscatter products with the following processing options:
```
Comment (Kampen1988) [inline Data/Sentinel1_processing_guide.qmd:58]: Please add `{target="_blank"}`

**Review by Kampen1988 — CHANGES_REQUESTED** [review]

File: Data/Sentinel1_processing_guide.qmd [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+    - Copernicus 30m DEM
+    - Copernicus 90m DEM 
+
+## [CloudFerro processing options](https://documentation.dataspace.copernicus.eu/APIs/On-Demand%20Production%20API.html){target="_blank"} 
+
+CloudFerro offers CARD-BS and COH6/COH12 products as a [service on demand](https://documentation.dataspace.copernicus.eu/APIs/On-Demand%20Production%20API.html){target="_blank"}: 
+
+- [Sentinel-1 (CARD-BS) BackScatter](https://documentation.dataspace.copernicus.eu/Data/Additional.html#sentinel-1-card-bs){target="_blank"}
+
+    - This processing option contains gamma0 geometric terrain correction (orthorectification), which is identical to the gamma0 (ellipsoid) backscatter coefficient with orthorectification option in Sentinel Hub processing options. No Radiometric Terrain Correction (RTC) is applied to this product.
```
Comment (megha-devaraju) [inline Data/Sentinel1_processing_guide.qmd:48]: @alek-cesarz Is the Copernicus DEM is used for the geometric terrain correction processing? If yes, which one?

File: Data/Sentinel1_processing_guide.qmd [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+CloudFerro offers CARD-BS and COH6/COH12 products as a [service on demand](https://documentation.dataspace.copernicus.eu/APIs/On-Demand%20Production%20API.html){target="_blank"}: 
+
+- [Sentinel-1 (CARD-BS) BackScatter](https://documentation.dataspace.copernicus.eu/Data/Additional.html#sentinel-1-card-bs){target="_blank"}
+
+    - This processing option contains gamma0 geometric terrain correction (orthorectification), which is identical to the gamma0 (ellipsoid) backscatter coefficient with orthorectification option in Sentinel Hub processing options. No Radiometric Terrain Correction (RTC) is applied to this product. 
+    - [Additional information](https://creodias.eu/eodata/sentinel-1/sentinel-1-l2-backscatter-bs/){target="_blank"}.
+
+- [Sentinel-1 (CARD-COH) Coherence](https://documentation.dataspace.copernicus.eu/APIs/On-Demand%20Production%20API.html){target="_blank"}
+
+    - The Sentinel-1 CARD COH (Copernicus Analysis Ready Data Coherence) processor generates a Sentinel-1 Level 2 product describing the coherence of a pair of images - 12 days apart. Concurrently, a terrain-correction (using DEM) is performed.
```
Comment (megha-devaraju) [inline Data/Sentinel1_processing_guide.qmd:53]: @alek-cesarz Can you provide information on the DEM that is used for the processing?

**Review by Kampen1988 — COMMENTED** [review]
Good to merge after confirmation from CloudFerro (@alek-cesarz ).

**Review by Kampen1988 — APPROVED** [review]

File: Data/Sentinel1_processing_guide.qmd (hunk 7 above)
Comment (alek-cesarz) [reply Data/Sentinel1_processing_guide.qmd:48]: 1.i. No, there is no preprocessed archive of CARD-BS products
1.ii. on-demand processing applies to products which are not already processed. these may be both over Europe and rest of the world
2. By default: COPDEM-30

File: Data/Sentinel1_processing_guide.qmd (hunk 8 above)
Comment (alek-cesarz) [reply Data/Sentinel1_processing_guide.qmd:53]: same ad for CARD-BS : COPDEM-30 (Copernicus DEM GLO-30)

**Review by Kampen1988 — APPROVED** [review]

**Pull request closed without merging**